README.md

```diff
@@ -514,7 +514,7 @@ Thus when you create a TODO, it is almost always your name that is given.
      end
      ```
 
-* <a name="no-default-args"></a>Do not use default arguments. Use an options
+* <a name="no-default-args"></a>Do not use default positional arguments. Use named arguments or an options
```

Comment:
@kmsun07 i agree that if named args in ruby 2+ are equivalent in perf to an options hash, and there's no other issues, than "use in ruby 2+" is fine with me. if there's any other quirks or caveats, then i would say that yes, we shouldn't use them at all prior to ruby 2.2 (which i assume fixes those quirks).